resources/views/calendars/events/_form.blade.php

```diff
@@ -12,7 +12,9 @@
         </span>
     </x-grid>
 @else
-    @include('cruds.fields.entity_link', ['entity' => $entityEvent->entity])
+    <div class="field-entity">
```

Comment:
![image](https://github.com/owlchester/kanka/assets/3263440/7f5c68aa-8dc0-464c-8bc8-c8efabbab540)
Double "field-entity" block, but it does need a `mb-5` class to add some bottom margin to make it look aligned with the rest of the form